src/components/common/dropdown/Dropdown.component.tsx

```diff
@@ -0,0 +1,37 @@
+import React, { useState } from 'react';
+import { faSortDown } from '@fortawesome/free-solid-svg-icons';
+import { DropdownIcon, DropdownItem, DropdownItemsContainer, SearchDropdownButton } from './Dropdown.styled-components';
+import { DropdownConfigItem } from './DropdownItemConfig';
+import PropTypes from 'prop-types';
+
+const DropdownComponent = ({itemsList, selected, changeSelected}) => {
+  const [isOpnened, updateVisability] = useState(false);
```

Comment:
```suggestion
  const [isOpened, updateVisibility] = useState(false);
```
ты можешь установить расширение для VSCode которое будет проверять орфграфию, в WebStorm это из коробки